docs/6-operating-guidelines.md

```diff
@@ -24,39 +24,41 @@ Antifragility and non-gameability emerge from simple frameworks. This limits the
   - `timeWeighting` can be either 1.0 for full-time or .5 for part-time contributors
   - Existing contributors get "diluted" as newcomers come in
   - Continuing contributors get additional weight per month they are active. This means historic contributors don't maintain their split weighting if they leave protocol development
-  - Anyone who stops contributing is removed from the split via periodic curation reviews
+  - Anyone who stops contributing should remove themselves from the membership
```

Comment:
I liked the old phrasing, it's fine to encourage self reporting, but anyone that stops contributing should be removed regardless if they remove themselves

Reply:
I think we need both.  I see an argument that can be made that the only way to exit is self-removal if this remains as is.

```suggestion
- Anyone who stops contributing should remove themselves from the membership, and may be removed via periodic curation reviews
```

Reply:
this works for me